jwql/tests/test_dark_monitor.py

```diff
@@ -70,6 +72,8 @@ def test_get_metadata():
     assert monitor.frame_time == 10.5
 
 
+@pytest.mark.skipif(os.path.expanduser('~') == '/home/jenkins',
```

Comment:
Can the Jenkins server not query MAST for some reason? I don't understand why this needs to be skipped on Jenkins. Information from the config.json file isn't used, and it doesn't rely on access to central store.

Reply:
@bhilbert4 See https://ssbjenkins.stsci.edu/blue/organizations/jenkins/STScI%2Fjwql/detail/PR-373/12/pipeline/21#step-39-log-19

I think the `dark_monitor` tests are failing because the tests need to instantiate the `Dark()` class (i.e. `dark_monitor.Dark(testing=True)`), and in doing so, needs to configure logging (since `Dark()` is decorated with `log_info` and `log_fail`), and since `logging_functions` need access to the config file, it ultimately fails.

I will look into trying to find a way around this.